a11y-dialog.js

```diff
@@ -236,7 +249,8 @@ A11yDialog.prototype._fire = function (type, event) {
 A11yDialog.prototype._bindKeypress = function (event) {
   // This is an escape hatch in case there are nested dialogs, so the keypresses
   // are only reacted to for the most recent one
-  if (!this.$el.contains(document.activeElement)) return
```

Comment:
Now that the dialog are properly nested, this escape hatch no longer works since the active element is *always within* all the “higher” dialogs.

We need to flip that logic and check whether the active element is *directly within* the current dialog.